src/components/ContentNode.vue

```diff
@@ -32,6 +33,7 @@ const BlockType = {
   termList: 'termList',
   unorderedList: 'unorderedList',
   dictionaryExample: 'dictionaryExample',
+  small: 'small',
```

Comment:
Shouldn't this be an inline element?

Reply:
I think it should, but in the RenderJSON draft I got it was specified as a block-level element, which is weird right?

@ethan-kusters can you look into this?

Reply:
I think we should consider changing it to be inline, to better match similar inline elements like emphasis/bold/etc.

Reply:
Right, but I think the node itself is emitted as a block one, so that might be a change DocC has to do too?

Reply:
Unfortunately I think this needs to be a block element because we don't support authoring inline directives.

Bold/emphasis/etc can be inline because we have built in syntaxes for them but directive-authored syntax will always be at the block-level.

Reply:
Got it. Sounds like it was a purposeful decision due to that limitation. Thanks @ethan-kusters!